sycl/test-e2e/lit.cfg.py

```diff
@@ -394,25 +394,22 @@
             dev_sg_sizes.append(sg_sizes_str.strip().split(' '))
 
     if dev_aspects == []:
-        lit_config.error('Cannot detect device aspect for {}\nstdout:\n{}\nstderr:\n'.format(
+        lit_config.error('Cannot detect device aspect for {}\nstdout:\n{}\nstderr:\n{}'.format(
             sycl_device, sp.stdout, sp.stderr))
         dev_aspects.append(set())
-    else:
-        # We might have several devices matching the same filter in the system.
-        # Compute intersection of aspects.
-        aspects = set(dev_aspects[0]).intersection(*dev_aspects)
-        lit_config.note('Aspects for {}: {}'.format(sycl_device, ', '.join(aspects)))
+    # We might have several devices matching the same filter in the system.
+    # Compute intersection of aspects.
+    aspects = set(dev_aspects[0]).intersection(*dev_aspects)
```

Comment:
Why are you removing the `else:`? Won't it result in python exception in `dev_aspects[0]` in this line?

Reply:
No because if `dev_aspects` is empty, we're already appending an empty `set()` on line 399.